ReactNativeClient/lib/components/screens/note.js

```diff
@@ -750,7 +756,6 @@ class NoteScreenComponent extends BaseScreenComponent {
 		if (this.state.mode === 'view') fieldToFocus = '';
 
 		if (fieldToFocus === 'title') this.refs.titleTextField.focus();
-		if (fieldToFocus === 'body') this.refs.noteBodyTextField.focus();
```

Comment:
Any reason this was removed? This is to focus the right field (either title or body) depending on user settings.